var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -359,43 +318,17 @@ def setup_dependent_package(self, module, dependent_spec):
         ]
 
     def with_or_without_verbs(self, activated):
```

Comment:
We can probably get rid of these functions how that things are simpler

Reply:
We run them like this:
https://github.com/spack/spack/blob/7449bf8bab94315f02ee200c9b65363edb7134f7/var/spack/repos/builtin/packages/openmpi/package.py#L472-L474
https://github.com/spack/spack/blob/7449bf8bab94315f02ee200c9b65363edb7134f7/var/spack/repos/builtin/packages/openmpi/package.py#L483-L484
and the values of the variants differ from the names of the required packages most of the time.